filament/backend/src/android/ExternalStreamManagerAndroid.cpp

```diff
@@ -39,7 +39,7 @@ static void loadSymbol(T*& pfn, const char *symbol) noexcept {
 
 ExternalStreamManagerAndroid& ExternalStreamManagerAndroid::get() noexcept {
     // declaring this thread local, will ensure it's destroyed with the calling thread
-    static UTILS_DECLARE_TLS(ExternalStreamManagerAndroid) instance;
+    static thread_local ExternalStreamManagerAndroid instance;
```

Comment:
What happens now if `thread_local` isn't supported?

Reply:
We're only using it on Android, where it's supported now. So I think it doesn't matter much (?)